### ISSUE TITLE
Head Tracking

### ISSUE THREAD

**Issue by saechisaechi** [endink/Mediapipe4u-plugin#65]
### MediaPipe4U Version

M4U_5_0_20230409

### UE Version

5.0.x

### UnrealEngine Type

Install From Epic Games Launcher

### What happened?

I am able to use livelink facial expressions with media4u just fine, but I'm wondering if it's possible to use the head tracking from Live Link as well?

I've tried locking the head from the blue print, but the head becomes stiff instead of taking data from live link. It only does facial capture. 

The head movement from media4u is amazing, but due to camera placement, it looks awkward because my head is always looking up instead of forward. I'd like to be able to use Livelink head tracking because the webcam position is best to capture the upper body but not the head.

**Comment by endink** [comment]
If you mean capturing the head from a mobile device, this is possible, but it need to write some math code on Android, I am busy with lipsync and offline tts feature development, so more features for device will be delayed, the important thing is that the M4U remoting receiver is a paid feature, free licensing does not package this feature(Only for UE editor).

**Comment by saechisaechi** [comment]
Yes, I have tried the head solver, and it works amazingly, but my webcam has to be on the table to capture my body well. It accurately tracks my head looking up, which I dont want. 

I have tried locking the head so that it receives head rotation from live link, but it's not working. Is it supposed to automatically receive head rotation from apple live link, or do I have to do it manually? 

I have a recording below to show that with lock head, it doesnt receive head rotation but receives face expressions. 

https://github.com/endink/Mediapipe4u-plugin/assets/87157576/49e36107-e8b5-4dca-8a0f-367b96b3dcb6
